content/influxdb/cloud/tools/downsample-data-quix.md

```diff
@@ -22,24 +22,22 @@ in [Quix Cloud](https://quix.io/) with a free trial.
 A common practice when processing high volume data is to downsample it before comitting 
```

Comment:
I don't see this topic where I expected it "cloud/tools". Very confusing.

![image](https://github.com/quixio/partner-docs-influxdb/assets/25511700/099c9d3d-4ff0-4083-a5fc-15a83e038862)

Reply:
hopefully dealt with